Polyfills/Canvas/Source/MeasureText.cpp

```diff
@@ -20,8 +20,10 @@ namespace Babylon::Polyfills::Internal
             env,
             JS_CONSTRUCTOR_NAME,
             {
-                InstanceAccessor("width", &MeasureText::GetWidth, &MeasureText::SetWidth),
-                InstanceAccessor("height", &MeasureText::GetHeight, &MeasureText::SetHeight),
+                InstanceAccessor("width", &MeasureText::GetWidth, nullptr),
+                InstanceAccessor("height", &MeasureText::GetHeight, nullptr),
+                InstanceAccessor("actualBoundingBoxLeft", &MeasureText::ActualBoundingBoxLeft, nullptr),
+                InstanceAccessor("actualBoundingBoxRight", &MeasureText::ActualBoundingBoxRight, nullptr)
```

Comment:
Can we just use regular properties with values instead of accessor functions? Accessor functions are way slower, and all they are doing here is returning a value, so just putting that value directly into a js property eliminates that overhead.

Reply:
Fixed.